spacelift/internal/testhelpers/helpers.go

```diff
@@ -207,3 +207,61 @@ func SetEquals(name string, values ...string) AttributeCheck {
 		return nil
 	}
 }
+
+// SetContains checks the set contains all specified values
+func SetContains(name string, values ...string) AttributeCheck {
```

Comment:
these three set functions are almost fully duplicated, is that a problem?

Reply:
Not a problem, no.